tests/RestApiSuite/TestGetAttributeFromLastReponse/content.txt

```diff
@@ -0,0 +1,7 @@
+!|post    |${URL}/users        |
+|FirstName|LastName|IsMafia|Age|
+|Tony     |Soprano |True   |22 |
+
+-!|script|RestTools|
+|check|get attribute|LastName|from last response|Soprano|
```

Comment:
I was confused by this test as it posts data and then check result of last response - which means it check result of this post